ChangeLog

```diff
@@ -13,6 +13,7 @@ Bugfix
      Found by redplait #590
    * Add MBEDTLS_MPI_CHK to check for error value of mbedtls_mpi_fill_random.
      Reported and fix suggested by guidovranken in #740
+   * Log correct number of ciphersuites used in Client Hello message. Fix for #918.
```

Comment:
After @andresag01 suggestions, I'd prefer a more specific entry here like "Modify debug output of ssl_write_client_hello to not consider SCSV entries in the ciphersuite count but to log them separately.".

Reply:
I personally do not mind the ChangeLog entry. Actually, I think that the change is so small that it probably should not have a ChangeLog entry...

Reply:
What about keeping it but moving it to the Changes section in it's extended form?

Reply:
I agree with Andres, that this is a minor change, only on a log message, which doesn't require a ChangeLog Entry